site/templates/application/google.jet

```diff
@@ -1,24 +1,19 @@
-{{block googleGa4Script(analyticsId=config("google_analytics_id"))}}
-  {{if hasPrefix(analyticsId,"G")}}
-  <!-- GA4 script if G tracking code is provided -->
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{analyticsId}}"></script>
-  {{end}}
-{{end}}
-
 {{block googleScripts(tagManagerId=config("google_tag_manager_id"), analyticsId=config("google_analytics_id"))}}
   <!-- Google integration scripts -->
   <script>
     function loadGoogleTagManager() {
-      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','{{ tagManagerId }}')
-    }
+      (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});
+      var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';
+      j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+      })(window,document,'script','dataLayer','{{ tagManagerId }}')}
```

Comment:
I know this has already merged, but that last `}` should be a new line since it's not part of the IIFE, it's the closing brace for the `function loadGoogleTagManager() {`

Reply:
A part of me also wants the semicolon even though it's not strictly necessary since it's a single statement (the IIFE), e.g.:

```
  ... '{{ tagManagerId }}');
}
```